roles/common/tasks/ssh.yml

```diff
@@ -7,6 +7,7 @@
   template: src=etc/sshd_config dest=/etc/ssh/sshd_config owner=root group=root mode=0644
   notify:
     - reload-sshd
+  when: not hardening.enabled
```

Comment:
So openstack-ansible-security does a lineinfile substitutions in the existing sshd_config file. In case someone chooses not to enable openstack-ansible-security we put our templated sshd_config in there. I just left it in there but we can probably just take it out since we are going to assume that we run the openstack-ansible-security role